pkg/controllers/metrics/node/controller.go

```diff
@@ -106,7 +104,6 @@ func nodeLabelNames() []string {
 		sets.New(lo.Values(wellKnownLabels)...).UnsortedList(),
 		resourceType,
 		nodeName,
-		nodeProvisioner,
```

Comment:
Should this be `nodeNodePool` now?

Reply:
I intentionally left this out since we have this label in the well known labels. If we add it here, this panics for having duplicates. I'm happy to lo.Reject() the above call on L104 and add it back in here, but I think just having a comment here is better than having redundant code. Thoughts?

Reply:
Yep, that makes sense to me. IIRC, that's the reason why I added the nodeProvisioner in here separately to begin with.